### PR TITLE
Mysql update

### PR DESCRIPTION
**Pull request by iannazzi**
Hi everyone, thanks for the awesome repo.... new to pull requests so I thought I would add in something simple to make sure I followed everything correctly....

This took me a bit to figure out - go straight to the mysql prompt within the mysql container. I updated the docs with an example.

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -86,7 +86,7 @@ services:
             - ./logs/nginx/:/var/log/nginx
             - ./nginx/sites/:/etc/nginx/sites-available
         ports:
-            - "80:80"
+            - "81:80"
```
Comment (philtrep) [inline docker-compose.yml]: Remove this from PR

File: mysql.sh [hunk 2]
```diff
@@ -0,0 +1,2 @@
+#docker-compose exec workspace bash
```
Comment (philtrep) [inline mysql.sh]: Remove this from PR

File: mysql/startup [hunk 3]
```diff
@@ -1,4 +1,7 @@
 DROP USER IF EXISTS 'MYSQL_USER'; 
 CREATE USER 'MYSQL_USER'@'%'; 
-CREATE DATABASE IF NOT EXISTS MYSQL_DATABASE; 
-GRANT ALL ON MYSQL_DATABASE.* TO 'MYSQL_USER'@'%' IDENTIFIED BY 'MYSQL_PASSWORD';
+CREATE DATABASE IF NOT EXISTS MYSQL_DATABASE;
```
Comment (philtrep) [inline mysql/startup]: Remove this from PR

File: up.sh [hunk 4]
```diff
@@ -0,0 +1,2 @@
+#!/usr/bin/env bash
```
Comment (philtrep) [inline up.sh]: Remove this from PR

File: workspace.sh [hunk 5]
```diff
@@ -0,0 +1,2 @@
+#docker-compose exec workspace bash
```
Comment (philtrep) [inline workspace.sh]: Remove this from PR

**Comment by philtrep** [comment]
@iannazzi Thanks for the PR!

The documentation part is very much welcomed but the scripts would have to be discussed in [the CLI repository](https://github.com/laradock/cli).

The modifications on the `startup` and `docker-compose.yml` files are not necessary.

Please perform the requested changes and I'll be more than happy to merge this in 👍

**Comment by iannazzi** [comment]
yeah startup and docker-compose.yml  should not have come over... so I screwed up the pull request. Ahh I think I know what I did wrong, I committed those changes first then branched. Then I modified the readme doc, then I committed that. I just want to send that one commit. Let me try again...then I should have it down.
